docs/releases/5.0.txt

```diff
@@ -406,6 +419,9 @@ Miscellaneous
 
 * The ``ForeignObject.get_reverse_joining_columns()`` method is deprecated.
 
+* The ``ModelAdmin.log_deletion`` method and the ``LogEntryManager.log_action``
+  method is deprecated.
```

Comment:
Deprecated methods should raise deprecation warnings etc. Please follow our [deprecating a feature guidelines](https://docs.djangoproject.com/en/4.2/internals/contributing/writing-code/submitting-patches/#deprecating-a-feature).